test/runtests.jl

```diff
@@ -44,6 +44,7 @@ test_pattern_get_surface()
 pth = joinpath(dirname(dirname(@__FILE__)), "samples")
 fls = filter(str->endswith(str,".jl"), readdir(pth))
 for fl in fls
+    println(fl)
```

Comment:
i'm a little bit stuck. I get **very** strange signal(6) and signal(11) around line 15 of sample_imagestream which the input string (inlined png). From that i worked backwards and somehow got the impression, that copy_path is creating the problems, as initially i got (always) the signal(6) from a destroy(path) call (in libcairo) and that's the only where a path object is created. 
Now i got:
```
lobi@orange4:~/juliarepo$ ../julia05/julia 
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: http://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.5.0-rc3+0 (2016-08-22 23:43 UTC)
 _/ |\__'_|_|_|\__'_|  |  Commit e6f843b* (4 days old release-0.5)
|__/                   |  x86_64-linux-gnu

julia> Pkg.test("Cairo")
INFO: Testing Cairo
sample_arc.jl
sample_arc_negative.jl
sample_clip_image.jl
sample_clip.jl
sample_copy_path.jl
sample_curve_rectangle.jl
sample_curve_to.jl
sample_dash.jl
sample_fill_and_stroke2.jl
sample_fill_style.jl
sample_gradient.jl
sample_image.jl
sample_imagepattern.jl
sample_imagestream.jl
*** Error in `/home/lobi/julia05/usr/bin/julia': double free or corruption (out): 0x0000000002d9e1f0 ***

signal (6): Aborted
while loading /home/lobi/.julia/v0.5/Cairo/samples/sample_imagestream.jl, in expression starting on line 15
raise at /build/eglibc-3GlaMS/eglibc-2.19/signal/../nptl/sysdeps/unix/sysv/linux/raise.c:56
abort at /build/eglibc-3GlaMS/eglibc-2.19/stdlib/abort.c:89
__libc_message at /build/eglibc-3GlaMS/eglibc-2.19/libio/../sysdeps/posix/libc_fatal.c:175
malloc_printerr at /build/eglibc-3GlaMS/eglibc-2.19/malloc/malloc.c:4996 [inlined]
_int_free at /build/eglibc-3GlaMS/eglibc-2.19/malloc/malloc.c:3840
jl_gc_free_array at /home/lobi/julia05/src/gc.c:744 [inlined]
sweep_malloced_arrays at /home/lobi/julia05/src/gc.c:765 [inlined]
gc_sweep_other at /home/lobi/julia05/src/gc.c:1032 [inlined]
_jl_gc_collect at /home/lobi/julia05/src/gc.c:1792 [inlined]
```
and (within the same session)
```
julia> Pkg.test("Cairo")
INFO: Testing Cairo
sample_arc.jl
sample_arc_negative.jl
sample_clip_image.jl
sample_clip.jl
sample_copy_path.jl
sample_curve_rectangle.jl
sample_curve_to.jl
sample_dash.jl
sample_fill_and_stroke2.jl
sample_fill_style.jl
sample_gradient.jl
sample_image.jl
sample_imagepattern.jl
sample_imagestream.jl
julia: /home/lobi/julia05/deps/srccache/llvm-3.7.1/lib/IR/Value.cpp:175: void llvm::Value::setValueName(llvm::ValueName*): Assertion `HasName == Ctx.pImpl->ValueNames.count(this) && "HasName bit out of sync!"' failed.

signal (6): Aborted
while loading /home/lobi/.julia/v0.5/Cairo/samples/sample_imagestream.jl, in expression starting on line 15
raise at /build/eglibc-3GlaMS/eglibc-2.19/signal/../nptl/sysdeps/unix/sysv/linux/raise.c:56
abort at /build/eglibc-3GlaMS/eglibc-2.19/stdlib/abort.c:89
__assert_fail_base at /build/eglibc-3GlaMS/eglibc-2.19/assert/assert.c:92
__assert_fail at /build/eglibc-3GlaMS/eglibc-2.19/assert/assert.c:101
_ZN4llvm5Value12setValueNameEPNS_14StringMapEntryIPS0_EE at /home/lobi/julia05/usr/bin/../lib/libLLVM-3.7.1.so (unknown line)
_ZN4llvm5Value11setNameImplERKNS_5TwineE at /home/lobi/julia05/usr/bin/../lib/libLLVM-3.7.1.so (unknown line)
_ZN4llvm5Value7setNameERKNS_5TwineE at /home/lobi/julia05/usr/bin/../lib/libLLVM-3.7.1.so (unknown line)
_ZN4llvm8FunctionC2EPNS_12FunctionTypeENS_11GlobalValue12LinkageTypesERKNS_5TwineEPNS_6ModuleE at /home/lobi/julia05/usr/bin/../lib/libLLVM-3.7.1.so (unknown line)
Create at /home/lobi/julia05/usr/include/llvm/IR/Function.h:123 [inlined]
function_proto at /home/lobi/julia05/src/jitlayers.h:95
emit_jlcall at /home/lobi/julia05/src/codegen.cpp:2592
emit_call_function_object at /home/lobi/julia05/src/codegen.cpp:2662
emit_invoke at /home/lobi/julia05/src/codegen.cpp:2693 [inlined]
emit_expr at /home/lobi/julia05/src/codegen.cpp:3165
emit_function at /home/lobi/julia05/src/codegen.cpp:4694
jl_compile_linfo at /home/lobi/julia05/src/codegen.cpp:810
jl_compile_for_dispatch at /home/lobi/julia05/src/gf.c:1311
...
```

i need a better idea, what happens (are working debugger...)